### ISSUE TITLE
Covers for recordings and channel logos on channel info not shown

### ISSUE THREAD

**Issue by sundalon** [pipelka/roboTV#56]
I'm facing three issues on my Nvidia Shield (2017) running the latest master:
1. channel logos are shown in schedule view but not on the channel info, is this related to me using the wrong sizes for logos? I'm using srp.256x256-226x226.dark.on.reflection atm.
2. a lot of recording posters are not shown even when doing a search and manually assigning them.
3. some but not all epgimages are loaded from the given URL

May 15 12:00:08 htpc vdr: [7509] get schedule called for channel 46 'S19.2E-1-1043-12502' - Eurosport 1 HD
May 15 12:00:08 htpc vdr: [7509] set epg image (channelUid: 987450925, eventid: 1101409) 'http://10.205.1.150/epgimages/1090300_0.jpg' (contentid: 0)
May 15 12:00:08 htpc vdr: [7509] set epg image (channelUid: 987450925, eventid: 1101410) '' (contentid: 0)

root@htpc:~# ll /var/cache/vdr/epgimages/1101410*
lrwxrwxrwx 1 vdr vdr 37 Mai 10 03:52 /var/cache/vdr/epgimages/1101410_0.jpg -> ./images/5489d695e4b0c014706b2d3c.jpg*

root@htpc:~# grep epg /etc/vdr/plugins/robotv/robotv.conf
EpgImageUrl = http://10.205.1.150/epgimages/%d_0.jpg

**Comment by sundalon** [comment]
Sorted all out. Seems like robotv is somewhat picky when being started without enabling write permissions first.
I enabled that after the first start and somehow robotv was not able to save epgimages and posters to the shield. After reinstallation I added the rights first and the started the app and all was working. :-)

The only thing that bugged me was that a huge amount of recordings were missing posters so I had to add/download them manually.
